src/js/ajaxfetch.js

```diff
@@ -0,0 +1,14 @@
+class AjaxWheather {
+  constructor() {
+    this.apiKey = '5a5c4674ce9bd002f4df3e98b673df57';
+  }
+
+  async getWheather(city) {
```

Comment:
- [ ] It has error handling for promises or async/await ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

Please implement the management of errors on your API call, you can implement a try-catch statement.